packages/nocodb/Dockerfile

```diff
@@ -70,25 +72,27 @@ ENV LITESTREAM_S3_SKIP_VERIFY=false \
     NODE_ENV=production \
     PORT=8080
 
-# Install additional runtime dependencies
-RUN apt-get update && apt-get install -y \
-    dumb-init \
+# Install runtime dependencies
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates `# required by wget for HTTPS support` \
     curl \
-    wget \
+    dumb-init \
+    wget `# for convenience of downstream users` \
     && curl -L "https://github.com/TomWright/dasel/releases/download/v2.8.1/dasel_linux_$(dpkg --print-architecture)" -o /usr/local/bin/dasel \
     && chmod +x /usr/local/bin/dasel \
+    && apt-get purge curl \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # Copy litestream binary and config file
-COPY --link --from=lt-builder /usr/src/lt /usr/local/bin/litestream
+COPY --link --from=lt-builder /usr/local/bin/litestream /usr/local/bin/litestream
 COPY --link ./docker/litestream.yml /etc/litestream.yml
 
 # Copy production code & main entry file
-COPY --link --from=builder /usr/src/app/ /usr/src/app/
+COPY --link --from=builder /usr/src/app/      /usr/src/app/
```

Comment:
remove the unnecessary whitespace here

Reply:
whitespace was for visual alignment; removed in https://github.com/nocodb/nocodb/pull/9756/commits/d16d3b1afe60fc6d59a37283f200cb7e0730b868